src/granite/exceptions.cr

```diff
@@ -3,7 +3,7 @@ module Granite
     getter model : Granite::Base
 
     def initialize(class_name : String, model : Granite::Base)
-      super("Could not process #{class_name}")
+      super("Could not save #{model.errors.map(&.to_s).join(", ")}")
```

Comment:
Tests are failing and the message is more cryptic than before IMO.
```
      Expected: "Could not process Parent"
                got: "Could not save Name Name cannot be blank"
```
Maybe `"Could not save #{class_name}. #{model.errors.map(&.to_s).join(", ")}" would be more clear.